### PR TITLE
Add Navbar

### PR DESCRIPTION
**Pull request by Kimboli**
hi donjar

### CODE REVIEW

**Comment by donjar** [comment]
1. Update from master
2. Check Travis
3. Use present tense verb in front

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -11,6 +11,60 @@
   </head>
 
   <body>
+
+ <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+
+    <a class="navbar-brand" href="http://nussucommit.com">
+      <img src="DSCF4500.jpg" width="70" height="70" alt="">NUSSU CommIT
```
Comment (donjar) [inline app/views/layouts/application.html.erb]: Use `image_tag`

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -11,6 +11,60 @@
...
+    </a>
+
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#NavbarContent" aria-controls="NavbarContent" aria-expanded="false" aria-label="Toggle navigation">
+    </button>
+
+    <div class="collapse navbar-collapse" id="NavbarContent">
+      <ul class="navbar-nav">
+      
+        <li class="nav-item ">
+          <a class="nav-link" href="#######">Home <span class="sr-only">(current)</span></a>
```
Comment (donjar) [inline app/views/layouts/application.html.erb]: What's with these `#####`... hahaha

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -11,6 +11,60 @@
...
+      <img src="DSCF4500.jpg" width="70" height="70" alt="">NUSSU CommIT
+    </a>
+
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#NavbarContent" aria-controls="NavbarContent" aria-expanded="false" aria-label="Toggle navigation">
+    </button>
+
+    <div class="collapse navbar-collapse" id="NavbarContent">
+      <ul class="navbar-nav">
+      
+        <li class="nav-item ">
```
Comment (donjar) [inline app/views/layouts/application.html.erb]: Extra space at the back

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -11,6 +11,60 @@
...
+
+        <li class="nav-item">
+          <a class="nav-link" href="#">Logout</a>
+        </li>
+
+      </ul>
+    </div>
+
+  <form class="form-inline my-2 my-lg-0">
+      <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
```
Comment (donjar) [inline app/views/layouts/application.html.erb]: Mind your indentation

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -11,6 +11,60 @@
...
+ <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+
+    <a class="navbar-brand" href="http://nussucommit.com">
+      <img src="DSCF4500.jpg" width="70" height="70" alt="">NUSSU CommIT
+    </a>
+
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#NavbarContent" aria-controls="NavbarContent" aria-expanded="false" aria-label="Toggle navigation">
+    </button>
+
+    <div class="collapse navbar-collapse" id="NavbarContent">
```
Comment (donjar) [inline app/views/layouts/application.html.erb]: Change to `navbar-content` for the ID. And change all references of `NavbarContent`

**Comment by Kimboli** [comment]
asides from the small change. Everything should be ok right? Pls check

**Comment by indocomsoft** [comment]
@Kimboli tidied everything up for you

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -11,6 +11,60 @@
   </head>
 
   <body>
-    <%= yield %>
+
+ <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```
Comment (donjar) [inline app/views/layouts/application.html.erb]: Fix the indentation here, make sure it's indented one more from body

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -11,6 +11,60 @@
...
+
+    <a class="navbar-brand" href="http://nussucommit.com">
+    <img src="http://i64.tinypic.com/24cuyr7.png" width="150" height="70" alt="nussucommit">
```
Comment (donjar) [inline app/views/layouts/application.html.erb]: Download the image, save in `app/assets/images`, then use `image_tag` here

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -11,6 +11,60 @@
...
+    </a>
+
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar-content" aria-controls="navbar-content" aria-expanded="false" aria-label="Toggle navigation">
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbar-content">
+    <ul class="navbar-nav">
+      
+        <li class="nav-item ">
```
Comment (donjar) [inline app/views/layouts/application.html.erb]: Extra space here?

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -11,6 +11,60 @@
...
+        <a class="nav-link" href="http://nussucommit.com">Home <span class="sr-only">(current)</span></a>
+        </li>
+      
+       	<li class="nav-item">
+        <a class="nav-link" href="#########">Grab Duty</a>
```
Comment (donjar) [inline app/views/layouts/application.html.erb]: Some of these have the correct links already, please link it properly :)

You can check the routes by using `bin/rails routes` in terminal, or just open something like `localhost:3000/asdf` lol

File: app/views/layouts/application.html.erb [hunk 10]
```diff
@@ -11,6 +11,60 @@
...
+        </li>
+
+        <li class="nav-item">
+        <a class="nav-link" href="#">Logout</a>
+        </li>
+
+     </ul>
+    </div>
+
+  <form class="form-inline my-2 my-lg-0">
```
Comment (donjar) [inline app/views/layouts/application.html.erb]: Haha I don't think we need search right

**Comment by donjar** [comment]
One more thing @Kimboli please rename the PR title so that it starts with a present tense verb. :)

**Comment by donjar** [comment]
@Kimboli I think the navbar is a bit too big ._. maybe can make it smaller a bit?

File: app/views/devise/sessions/new.html.erb [hunk 11]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (donjar) [inline app/views/devise/sessions/new.html.erb]: Remove unneccessary spaces

File: app/views/devise/sessions/new.html.erb [hunk 12]
```diff
@@ -38,5 +40,11 @@
     </div>  
   </div>
 
+ 
+  <% content_for :page_attributes do %>
```
Comment (donjar) [inline app/views/devise/sessions/new.html.erb]: Mind your indentation

File: app/views/layouts/application.html.erb [hunk 13]
```diff
@@ -10,7 +10,72 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
-  <body>
-    <%= yield %>
+  <body<%= yield :page_attributes %>>
```
Comment (donjar) [inline app/views/layouts/application.html.erb]: Please compress this file. Don't sprinkle whitespace everywhere

File: app/views/users/edit.html.erb [hunk 14]
```diff
@@ -35,4 +35,6 @@
     <%= f.submit "Save"%>
   </p>
 
+
```
Comment (donjar) [inline app/views/users/edit.html.erb]: There's extra whitespace here too

File: app/assets/stylesheets/application.scss [hunk 15]
```diff
@@ -21,3 +21,5 @@
 @import 'duties';
 @import 'availabilities';
 @import 'announcements';
+
+body[no-nav] nav{display: none;}
```
Comment (donjar) [inline app/assets/stylesheets/application.scss]: Enter it out

File: app/views/users/edit.html.erb [hunk 16]
```diff
@@ -34,5 +34,5 @@
   <p>
     <%= f.submit "Save"%>
   </p>
-
+
```
Comment (donjar) [inline app/views/users/edit.html.erb]: Remove this

File: app/assets/stylesheets/application.scss [hunk 17]
```diff
@@ -14,10 +14,19 @@
 @import 'bootstrap';
 
 @import 'colors';
```
Comment (donjar) [inline app/assets/stylesheets/application.scss]: Why space out this file?

File: app/assets/stylesheets/application.scss [hunk 18]
```diff
@@ -14,10 +14,19 @@
...
+
 @import 'profile';
+
 @import 'duties';
+
 @import 'availabilities';
+
 @import 'announcements';
+
+body[no-nav] nav{display: none;}
```
Comment (donjar) [inline app/assets/stylesheets/application.scss]: And, you should also properly format this line

**Pull request merged**